### PR TITLE
19513 - Add more error catching for csv feedback reconciliation

### PR DESCRIPTION
**Pull request by Jxio**
*Issue #:*
https://github.com/bcgov/entity/issues/19513
*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-pay license (Apache 2.0).

### CODE REVIEW

**Review by seeker25 — COMMENTED** [review]
re-raise exception, so it stops the flow

**Comment by Jxio** [comment]
it should catch other error like syncing credit records
image

File: pay-queue/src/pay_queue/services/payment_reconciliations.py [hunk 1]
```diff
@@ -251,20 +253,39 @@ def _process_file_content(content: str, cas_settlement: CasSettlementModel,
...
 
         # Commit the transaction and process next row.
         db.session.commit()
 
     # Create payment records for lines other than PAD
-    _create_payment_records(content)
+    try:
+        _create_payment_records(content)
+    except Exception as e: # NOQA # pylint: disable=broad-except
+        error_msg = f'Error creating payment records: {str(e)}'
```
Comment (seeker25) [inline pay-queue/src/pay_queue/services/payment_reconciliations.py:267]: re-raise the exception here

**Review by seeker25 — CHANGES_REQUESTED** [review]
edit, didn't see return

**Review by seeker25 — APPROVED** [review]

**Pull request merged**